Student/StudentE2ETests/Announcements/DSAnnouncementsE2ETests.swift

```diff
@@ -41,4 +41,38 @@ class DSAnnouncementsE2ETests: E2ETestCase {
         XCTAssertEqual(DiscussionDetails.title.label(), announcementTitle)
         XCTAssertTrue(app.find(label: announcementMessage).exists())
     }
+
+    @available(iOS 15, *)
+    func testAnnouncementToggleE2E() {
+        let student = seeder.createUser()
+        let teacher = seeder.createUser()
+        let course = seeder.createCourse()
+        seeder.enrollTeacher(teacher, in: course)
+        seeder.enrollStudent(student, in: course)
+
+        let dateFormatter = ISO8601DateFormatter()
+        let globalAnnouncementSubject = "This is a GA"
+        let globalAnnouncementMessage = "This will disappear in 4 minutes"
+        let globalAnnouncementStartAt = dateFormatter.string(from: Date.now.addMinutes(-1))
+        let globalAnnouncementEndAt = dateFormatter.string(from: Date.now.addMinutes(3))
```

Comment:
We can use `Date()` instead of `Date.now`. The result should be the same and we could remove the `@available(iOS 15, *)` statement.